src/store/modules/user/index.ts

```diff
@@ -0,0 +1,44 @@
+import UsersService from '../../../services/users.services';
+import { LOGIN } from '../../mutations-types';
+
+const state: UserState = {
+  user: null
+};
+const getters = {
+  getUser: (state: UserState) => state.user
+
+};
+const mutations = {
+  [LOGIN.CHECK_IN]: (state: UserState, user: User) => (state.user = user),
+  [LOGIN.CHECK_OUT]: (state: UserState) => { state.user = null; },
+};
+const actions = {
+  logIn: ({ commit }: any, { username }: { username: string; password: string }) => {
```

Comment:
I would prefer an explicit type. Instead of `{ username: string, password: string }` something like `LoginRequest` which itself is an interface with the required properties. I believe it produces less clutter and an easier to read code.